client/src/components/modal.js

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect } from 'react';
+import './modal.css';
+import $ from 'jquery';
+const Modal=() =>{
+
+    useEffect(() => {
+        function readyFn(){
+            document.body.classList.add("modal-is-open");
+        }
+        $( document ).ready(readyFn() );
+
+const modalBtn = document.querySelector(".js-btn");
+const modalClose = document.querySelectorAll(".jsModalClose");
+const modal = document.querySelector(".modal-overlay");
+
+modalBtn.addEventListener("click", () => {
+  document.body.classList.add("modal-is-open");
+});
+
+modalClose.forEach(item => {
+  item.addEventListener("click", () => {
+    document.body.classList.remove("modal-is-open");
+  });
+});
+// close modal when click outside
+modal.addEventListener("click", e => {
+  if (!e.target.closest(".modal")) {
+    document.body.classList.remove("modal-is-open");
+  }
+});
+      });
+    
+    return(
+        <>
+            <main className="main">
+   <div className="container">
+      <button className="btn js-btn">Notice</button>
+   </div>
+</main>
+<div className="modal-overlay js-modal">
```

Comment:
Improving the indentation will help other developers to understand the code better and quicker!